src/Exception/BunqException.php

```diff
@@ -7,10 +7,10 @@ class BunqException extends \Exception
 {
     /**
      * @param string $message
-     * @param string[] $args
+     * @param string[] $responseCode
      */
-    public function __construct($message, $args = [])
+    public function __construct($message, $responseCode = [])
```

Comment:
Please add strict types for both parameters; Also `$responseCode` shouldn't be equal to an array I believe :D